apps/server/src/user/dto/create-user.dto.ts

```diff
@@ -4,7 +4,7 @@ import { IsString } from 'class-validator';
 
 export class CreateUserDto {
   @IsString()
-  @ApiProperty()
+  @ApiProperty({ default: 'lftechnology.com' })
```

Comment:
lets not have leapfrog.com lets have user@mail.com

Reply:
Done